src/rp2_common/pico_stdio_usb/include/pico/stdio_usb.h

```diff
@@ -94,10 +104,29 @@ extern "C" {
 extern stdio_driver_t stdio_usb;
 
 /*! \brief Explicitly initialize USB stdio and add it to the current set of stdin drivers
- *  \ingroup pico_stdio_uart
+ *  \ingroup pico_stdio_usb
+ *
+ *  \ref PICO_STDIO_USB_POST_CONNECT_WAIT_DELAY_MS can be set to cause this method to wait for a CDC connection
```

Comment:
Should this be referring to `PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS` ?

Reply:
i removed this comment since we now have the other stuff